lib/nconf/stores/file.js

```diff
@@ -199,11 +202,31 @@ File.prototype.stringify = function (format) {
 File.prototype.parse = function (contents) {
   var parsed = this.format.parse(contents);
 
-  if (!this.secure) {
-    return parsed;
+  if (this.secure) {
+    var self = this;
+    var outdated = false;
+    parsed = Object.keys(parsed).reduce(function (acc, key) {
+      var value = parsed[key];
+      var decipher = crypto.createDecipher(value.alg, self.secure.secret);
```

Comment:
You're calling `crypto.createDecipher()` unconditionally here, then conditionally do the work again when an IV is detected. That means you're doing twice the work.

Worse, it'll emit warnings with Node.js versions where `crypto.createDecipher()` is deprecated (v11.x and up.)